recipes/rpclib/all/conanfile.py

```diff
@@ -0,0 +1,69 @@
+from conans import ConanFile, tools, CMake
+import os
+import glob
+
+
+class rpclibConan(ConanFile):
+    name = "rpclib"
+    description = "A modern C++ msgpack-RPC server and client library."
+    license = "MIT"
+    topics = ("rpc", "rpclib", "ipc")
```

Comment:
```suggestion
    topics = ("rpc", "ipc", "rpc-server")
```

Don't use project's name as topic.